content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -36,33 +36,45 @@ Pod:
 
 1. Create a Pod based on the YAML configuration file:
 
-        kubectl create -f https://k8s.io/examples/pods/inject/envars.yaml
+    ```shell
+    kubectl create -f https://k8s.io/examples/pods/inject/envars.yaml
+    ```
 
 1. List the running Pods:
 
-        kubectl get pods -l purpose=demonstrate-envars
+    ```shell
+    kubectl get pods -l purpose=demonstrate-envars
+    ```
 
     The output is similar to this:
 
-        NAME            READY     STATUS    RESTARTS   AGE
-        envar-demo      1/1       Running   0          9s
+    ```log
+    NAME            READY     STATUS    RESTARTS   AGE
+    envar-demo      1/1       Running   0          9s
+    ```
 
 1. Get a shell to the container running in your Pod:
 
-        kubectl exec -it envar-demo -- /bin/bash
+    ```shell
+    kubectl exec -it envar-demo -- /bin/bash
+    ```
 
 1. In your shell, run the `printenv` command to list the environment variables.
 
-        root@envar-demo:/# printenv
+    ```shell
+    root@envar-demo:/# printenv
+    ```
 
     The output is similar to this:
 
-        NODE_VERSION=4.4.2
-        EXAMPLE_SERVICE_PORT_8080_TCP_ADDR=10.3.245.237
-        HOSTNAME=envar-demo
-        ...
-        DEMO_GREETING=Hello from the environment
-        DEMO_FAREWELL=Such a sweet sorrow
+    ```log
+    NODE_VERSION=4.4.2
+    EXAMPLE_SERVICE_PORT_8080_TCP_ADDR=10.3.245.237
+    HOSTNAME=envar-demo
+    ...
+    DEMO_GREETING=Hello from the environment
+    DEMO_FAREWELL=Such a sweet sorrow
+    ```
```

Comment:
The same

Reply:
The Black Friday Markdown engine is very finicky about indentation. Four spaces are necessary in this case so we should use no more than that. Furthermore, it’s standard throughout the docs to use backtick-based code blocks and to indicate the appropriate language where possible.